lib/vfscore/automount.c

```diff
@@ -123,8 +155,33 @@ static int vfscore_rootfs(void)
 		uk_pr_crit("Failed to mount %s (%s) at /: %d\n", rootdev, rootfs, errno);
 		return -1;
 	}
+#endif /* CONFIG_LIBVFSCORE_AUTOMOUNT_ROOTFS */
+
+	return 0;
+}
+
+static int vfscore_volumes_automount(void)
```

Comment:
This is more a minor detail, more aesthetically: I'd prefer two functions at the same level:
`vfscore_automount_rootfs`
`vfscore_automount_volumes`
and a third function that calls these two and is registered to the init table.
btw, it is a clever idea to create empty function bodies depending on the configuration.